src/pages/HoloFuel/index.js

```diff
@@ -0,0 +1,8 @@
+// import Component from './HoloFuelTxOverview';
```

Comment:
The directory structure here needs a tweak. Each page should have its own directory `PageComponentName` with children `index.js`, `PageComponentName.js`, `PageComponentName.module.css` etc. I see the value in nesting them all under a master `HoloFuel` directory to keep them separate, but as we haven't figured out how we're organizing the HoloFuel specific code I think for now just having those directories in the top level is fine.

Reply:
Thanks for this clarification. :rocket:  This is one of the questions I had and wished to discuss together.